### DISCUSSION TITLE
Icepak Change Design Setting

### DISCUSSION THREAD

**Discussion by mchoi327** [ansys/pyaedt#4856] [Q&A] [answered]
Hi, it seems that 'change_design_settings' or 'edit_design_settings' have no attributes yet.

Is there a way to change the default Icepak design setting (in the attached figure), e.g., change the 'AmbientTemp' from '20cel' to '25cel', using the 'change_design_settings' method?

![image](https://github.com/ansys/pyaedt/assets/122655322/de638e48-59b2-47a3-9429-04248ae67e5b)

**Comment by lorenzovecchietti** [comment, marked as answer]
Hi, you can use the design_settings object:
`ipk.design_settings["AmbTemp"]="25cel"`

You can interact with the object like a dictionary.
